tools/donate-cpu.py

```diff
@@ -281,8 +281,15 @@ def scan_package(work_path, cppcheck_path, jobs):
     cppcheck_cmd = cppcheck_path + '/cppcheck' + ' ' + options
     cmd = 'nice ' + cppcheck_cmd
     returncode, stdout, stderr, elapsed_time = run_command(cmd)
-    print('cppcheck finished with ' + str(returncode))
-    if returncode == -11 or stderr.find('Internal error: Child process crashed with signal 11 [cppcheckError]') > 0:
+    sig_num = -1
+    sig_msg = 'Internal error: Child process crashed with signal '
+    sig_pos = stderr.find(sig_msg)
+    if not sig_pos == -1:
```

Comment:
```suggestion
    if sig_pos != -1:
```
IMHO is better readable.